test/protocol-buffer-conversion-test.ts

```diff
@@ -17,4 +38,76 @@ describe('Protocol Buffer Conversion', function(): void {
     assert.deepEqual(currency.code, currencyProto.getCode())
     assert.deepEqual(currency.name, currencyProto.getName())
   })
+
+  it('Convert PathElement protobuf with all fields set to XRPPathElement', function(): void {
+    // GIVEN a PathElement protocol buffer with all fields set.
+    const pathElementProto = testPathElement
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const pathElement = new XRPPathElement(pathElementProto)
+
+    // THEN the currency converted as expected.
+    assert.equal(
+      pathElement.account,
+      pathElementProto.getAccount()!.getAddress(),
+    )
+    assert.deepEqual(
+      pathElement.currency,
+      new XRPCurrency(pathElementProto.getCurrency()!),
+    )
+    assert.equal(pathElement.issuer, pathElementProto.getIssuer()!.getAddress())
+  })
+
+  it('Convert PathElement protobuf with no fields set to XRPPathElement', function(): void {
+    // GIVEN a PathElement protocol buffer with no fields set.
+    const pathElementProto = new Payment.PathElement()
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const pathElement = new XRPPathElement(pathElementProto)
+
+    // THEN the currency converted as expected.
+    assert.isNull(pathElement.account)
+    assert.isNull(pathElement.currency)
+    assert.isNull(pathElement.issuer)
+  })
+
+  // MARK: - Org_Xrpl_Rpc_V1_Transaction.Path
+  it('Convert Path protobuf with no paths to XRPPath', function(): void {
+    // GIVEN a set of paths with zero paths.
+    const pathProto = new Payment.Path()
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const path = new XRPPath(pathProto)
+
+    // THEN there are zero paths in the output.
+    assert.equal(path.pathElements.length, 0)
+  })
+
+  it('Convert Path protobuf with one Path to XRPPath', function(): void {
+    // GIVEN a set of paths with one path.
+    const pathProto = new Payment.Path()
+    pathProto.addElements(testPathElement)
```

Comment:
Can we not do `new Payment.Path(testPathElement)`?

Reply:
the Protobuf constructors don't take arguments - they use getters and setters to interact with all the fields after object construction